packages/teleterm/src/ui/DocumentGateway/DocumentGateway.tsx

```diff
@@ -71,20 +71,55 @@ export function DocumentGateway(props: State) {
     );
   }
 
+  let cliConnectionString: string;
```

Comment:
In such cases I like to create a new function, so I can avoid mutations:
```typescript
function getCliConnectionString(gateway: Gateway) {
  switch (gateway.protocol as GatewayProtocol) {
    case 'mongodb':
      return useMongo(gateway);
    case 'postgres':
      return usePostgres(gateway);
    case 'mysql':
      return useMySql(gateway);
  }
}
``` 

And theoretically, we shouldn't use `use...` names as they're reserved for hooks and we can't use hooks conditionally (like in switch-case statements), but I'm fine with leaving it if we are going to remove them anyway.